pyproject.toml

```diff
@@ -49,7 +49,7 @@ liccheck = "^0.9.1"
 cmakelang = "^0.6.13"
 
 [build-system]
-requires = ["scikit-build-core", "oldest-supported-numpy", "fgen==0.3.1"]
+requires = ["scikit-build-core", "oldest-supported-numpy", "fgen[templates]==0.4.1"]
```

Comment:
@znicholls this is a nice part about splitting out the template dependencies. `fgen[templates]` is only needed as a build dep

Reply:
Nice